test/L2/paused/L2VotingPowerPaused.t.sol

```diff
@@ -33,6 +33,12 @@ contract L2VotingPowerPausedTest is Test {
 
     address lockingPositionContractAddress;
 
+    function assetInitParamsEq() internal {
```

Comment:
it should be called `assertInitParamsEq`

Reply:
updated